controllers/replica_set_controller.go

```diff
@@ -77,11 +82,6 @@ const (
 	// configured
 	lastVersionAnnotationKey    = "mongodb.com/v1.lastVersion"
 	lastSuccessfulConfiguration = "mongodb.com/v1.lastSuccessfulConfiguration"
-	// tlsRolledOutAnnotationKey indicates if TLS has been fully rolled out
-	tlsRolledOutAnnotationKey      = "mongodb.com/v1.tlsRolledOut"
-	hasLeftReadyStateAnnotationKey = "mongodb.com/v1.hasLeftReadyStateAnnotationKey"
-
-	trueAnnotation = "true"
```

Comment:
tls annotation checking is no longer required, this is now accounted for when determining the order of the sts/automation config updates.